kbs/src/api/src/attestation/coco/grpc.rs

```diff
@@ -118,6 +134,28 @@ impl Attest for GrpcClientPool {
 
         Ok(token)
     }
+
+    async fn generate_supplemental_challenge(
+        &self,
+        tee: Tee,
+        tee_parameters: String,
+    ) -> Result<String> {
+        let grpc_tee = to_grpc_tee(tee).unwrap();
```

Comment:
We can use
```
let tee = serde_json::to_string(&tee)
            .context("CoCo AS client: serialize tee type failed.")?
            .trim_end_matches('"')
            .trim_start_matches('"')
            .to_string();
```
similiarly to what we use in `verify()` without defining `to_grpc_tee()`.